### PR TITLE
fix: refactor owner to admin

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Change all owner texts to admin

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1236 #done

### CODE REVIEW

File: packages/shared/src/graphql/sources.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export interface Squad extends Source {
   permalink: string;
   public: boolean;
   type: SourceType.Squad;
-  owners?: string[];
+  admins?: string[];
```
Comment (capJavert) [inline packages/shared/src/graphql/sources.ts:45]: I think this is not used, `owners` and now `admins` does not exist as column or in `query Source` on API.

**Review by capJavert — CHANGES_REQUESTED** [review]
Lots of renaming, one thing caught my eye, other then that 👏

File: packages/shared/src/graphql/sources.ts (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/graphql/sources.ts]: Ah good point removed it now.

**Review by capJavert — APPROVED** [review]
🚀

**Pull request merged**